Casks/beekeeper-studio.rb

```diff
@@ -0,0 +1,14 @@
+cask 'beekeeper-studio' do
+  version '1.1.2'
+  sha256 '5850f254c5506ec65d39737200a1409c1e670c2a1644e7b1d72deaacf4d90f06'
+
+  # github.com/beekeeper-studio/beekeeper-studio was verified as official when first introduced to the cask
```

Comment:
Trailing slash is actually significant here:
```suggestion
  # github.com/beekeeper-studio/beekeeper-studio/ was verified as official when first introduced to the cask
```

Reply:
Thanks for the reviews & suggestions